fire/io/bernese/bernese.py

```diff
@@ -0,0 +1,314 @@
+        """
+        for linje_nr, line in enumerate(crd_data, start=1):
+            if linje_nr == 1:  # header med ugenr/comb week
+                self.resultatset.gnss_week = crd_parse_weekline(line)
+            elif linje_nr == 3:  # header med datum og epoke
+                headerlinje = crd_parse_epochdatumline(line)
+                self.resultatset.datum = headerlinje["DATUM"]
+                self.resultatset.epoch = strptime(
+                    headerlinje["EPOCH"], "%Y-%m-%d %H:%M:%S"
+                )
+            elif linje_nr >= 7:  # data begynder på linje 7 i CRD-filer
+                if not line.isspace():  # skip evt. tomme linjer
+                    datalinje = crd_parse_dataline(line)
+                    station = Station(
+                        navn=datalinje["STATION NAME"],
+                        flag=datalinje["FLAG"],
+                        koordinat=Koordinat(
+                            x=datalinje["X"], y=datalinje["Y"], z=datalinje["Z"]
+                        ),
+                        kovarians=None,
+                        spredning=None,
+                    )  # kovarians of spredning bliver sat senere
+                    self.resultatset.stations[datalinje["STATION NAME"]] = station
+
+    def cov_parse(self, cov_data: list) -> None:
+        """
+        Parse datalinjerne fra en kovarians (COV) fil.
+        """
+        linje_nr = 0
+        temp_table = {}
+        for station in self.resultatset.stations.keys():
+            temp_table[station] = {}  # opret en tabel med navne fra CRD parsing
+        for line in cov_data:
+            linje_nr = linje_nr + 1
+            if linje_nr == 1:
+                if cov_parse_weekline(line) != self.resultatset.gnss_week:
+                    raise ValueError("Forskellige GNSS uger i CRD og COV!")
+            if linje_nr >= 12:  # data begynder på linje 12 i COV-filer
+                if not line.isspace():  # skip evt. tomme linjer
+                    matrice = {}
+                    temp = cov_parse_dataline(line)
+                    if (
+                        temp["STATION 1"] == temp["STATION 2"]
+                    ):  # data er kun interessant hvis det er samme station
+                        koordinatpar = temp["XYZ 1"] + temp["XYZ 2"]
+                        # vi er nødt til at erstatte D med E pga. Fortran/Python forskelle i videnskabelig notation
+                        matrice[koordinatpar] = float(
+                            temp["MATRIX ELEMENT"].replace("D", "E")
+                        )
+                        temp_table[temp["STATION 1"]][koordinatpar] = matrice[
+                            koordinatpar
+                        ]
+        for entry in temp_table.keys():
+            if (
+                not temp_table[entry] == {}
+            ):  # ignorer stationer fra CRD vi ikke har kovarianser for i COV
+                self.resultatset.stations[entry].kovarians = Kovarians(
```

Comment:
Tre ting:

1. `not a == b` er det samme som `a != b`
2. Det er ikke nødvendigt at specificere `.keys()` når man itererer over en dict. Det kan selvfølgelig godt give mening ud fra et læsbarhedssynspunkt, om det er nødvendigt her vil jeg lade dig om at vurdere. 
3. brug `continue`:

```suggestion
        for entry in temp_table:
            if temp_table[entry] == {}:  # ignorer stationer fra CRD vi ikke har kovarianser for i COV
               continue
            self.resultatset.stations[entry].kovarians = Kovarians(
            ...
```

Reply:
Done :)